src/Microsoft.Azure.SignalR.Management/WebsocketsHubLifetimeManager.cs

```diff
@@ -13,10 +13,13 @@ namespace Microsoft.Azure.SignalR.Management
 {
     internal class WebSocketsHubLifetimeManager<THub> : ServiceLifetimeManagerBase<THub>, IHubLifetimeManagerForUserGroup where THub : Hub
     {
+        private readonly IServiceConnectionManager<THub> _serviceConnectionManager;
+
         public WebSocketsHubLifetimeManager(IServiceConnectionManager<THub> serviceConnectionManager, IHubProtocolResolver protocolResolver,
             IOptions<HubOptions> globalHubOptions, IOptions<HubOptions<THub>> hubOptions, ILoggerFactory loggerFactory) :
             base(serviceConnectionManager, protocolResolver, globalHubOptions, hubOptions, loggerFactory?.CreateLogger(nameof(WebSocketsHubLifetimeManager<Hub>)))
         {
+            _serviceConnectionManager = serviceConnectionManager;
```

Comment:
Lifetime is the application layer, `IServiceConnectionManager` is the transport layer, any advantages to having `IServiceConnectionManager` inside Lifetime?